pallets/minterest-protocol/src/lib.rs

```diff
@@ -626,6 +626,12 @@ impl<T: Config> Pallet<T> {
 
 		T::MultiCurrency::deposit(wrapped_id, &who, wrapped_amount)?;
 
+		liquidity_pools::PoolUserParams::<T>::mutate(underlying_asset, &who, |p| {
+			if !p.liquidation_attempts.is_zero() {
+				p.liquidation_attempts = u8::zero();
+			}
```

Comment:
I don`t think we will benefit from this check.
```suggestion
			p.liquidation_attempts = u8::zero();
```

Reply:
agree

Reply:
Guys, One write = 1 000 000 000 weight, it's almost 1 millisecond :)
What is disadvantage of this check?

Reply:
I believe if we are already in mutate that is a write

Reply:
Maybe it`s worth checking by generating weights for both cases)

Reply:
You were right, If we call mutate, then a write will be made.
What about this variant?

let mut pool_params = <LiquidityPools<T>>::pool_user_data(underlying_asset, &who);
if !pool_params.liquidation_attempts.is_zero() {
	pool_params.liquidation_attempts = u8::zero();
	liquidity_pools::PoolUserParams::<T>::insert(underlying_asset, &who, pool_params);
}

Reply:
Looks good for me